### PR TITLE
Creta emodal HOC.

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: src/hoc/ModalHOC.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ModalHOC extends React.Component {
```
Comment (plzen) [inline src/hoc/ModalHOC.js]: This class isn't HOC. It's just wrapper component. HOC it's a function which accepts another component as first argument and returns new component.
Also, usual naming for files it's `with...`, so in your case it should be like `withModal` for file and `WithModal` for component.

Comment (plzen) [reply src/hoc/ModalHOC.js]: https://reactjs.org/docs/higher-order-components.html

Comment (plzen) [reply src/hoc/ModalHOC.js]: https://reactjs.org/docs/render-props.html

Comment (pink-bunny) [reply src/hoc/ModalHOC.js]: Fixed.

**Pull request merged**